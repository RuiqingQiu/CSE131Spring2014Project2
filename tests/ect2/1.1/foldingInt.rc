const auto a = 0x1F;
static const auto b = a - 012;
const int c = a * 5;
static const int d = a + b + c;

function : int main(){
   const auto e = d / a;
   static const auto f = e % 4;
   const int g = e | a;
   static const int h = g & 0x1;

   cout << a << ", " << b << ", " << c << ", " << d << ", ";
   cout << e << ", " << f << ", " << g << ", " << h << ", ";
   cout << e^a << ", " << e^e << ", " << -1|e << ", " << e&0 << ", ";
   cout << -a << ", " << +a << ", " << a-b-(c+d-e) << ", " << d/a/f << endl;

   return 0;
}
