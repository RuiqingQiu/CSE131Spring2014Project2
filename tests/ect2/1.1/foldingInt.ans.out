31, 21, 155, 207, 6, 2, 31, 1, 25, 0, -1, 0, -31, 31, -346, 3
